client/src/Frontend/Components/ArtifactImage.tsx

```diff
@@ -5,7 +5,7 @@ import styled, { css } from 'styled-components';
 import dfstyles from '../Styles/dfstyles';
 
 export const ARTIFACT_URL = 'https://d2wspbczt15cqu.cloudfront.net/v0.6.0-artifacts/';
```

Comment:
Again not relevant to specific PR, but seems odd to have a dependency on a random server built by DF. Can we download these assets or access this code?

Reply:
I'd like to switch the assets to ipfs. WDYT?

Reply:
Yes, we should do it